cmd/kube-apiserver/app/server.go

```diff
@@ -199,7 +199,7 @@ func (s *APIServer) Run(_ []string) error {
 		glog.Fatalf("Failure to start kubelet client: %v", err)
 	}
 
-	_, v1beta3 := s.RuntimeConfig["api/v1beta3"]
+	_, disableV1beta3 := s.RuntimeConfig["noapi/v1beta3"]
```

Comment:
Just use api/v1beta3. Flags are an API to cluster turnup tools/scripts. We basically can't change them, and there's no good reason to here. ConfigurationMap supports values. We should just parse out true/false.
https://github.com/GoogleCloudPlatform/kubernetes/blob/aaa4a354d40fccd3c4ed1f3d6cb9ecea1e2f407f/pkg/util/configuration_map.go#L30